world_frame: "map"
odom_frame: "odom"
rover_frame: "base_link"
use_odom_frame: false

gps_linearization:
  reference_point_latitude: 42.3012279
  reference_point_longitude: -83.6951626
  reference_point_altitude: 234.1
  reference_heading: 90.0
